### PR TITLE
applying wallpaper sent by meilix generator

### LINKED ISSUE #443: Apply wallpaper sent by meilix-generator
- Issue type: Bug report

### Short description

uploaded wallpaper in meilix-generator is not applied in the custom ISO

### Expected behaviour
wallpaper should be applied to the custom ISO

### Actual behaviour

wallpaper URL sent by meilix-generator is not being handled by the build scripts

### PR DESCRIPTION
**Pull request by alok760**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Fixes #443

### CODE REVIEW

**Comment by xeon-zolt** [comment]
@pranav1698 what he did not followed ?

File: scripts/chroot_reconst.sh [hunk 1]
```diff
@@ -107,6 +107,11 @@ setterm -blank 0 >> /etc/issue
 # Meilix default settings
 dpkg -i --force-overwrite meilix-default-settings_1.0_all.deb
 
+if [ "$wallpaper_url" != "" ]
+then
+      wget $wallpaper_url -O "/usr/share/lxqt/themes/meilix/wallpaper.jpg"
```
Comment (xeon-zolt) [inline scripts/chroot_reconst.sh]: quite flag to reduce logs !!

**Comment by xeon-zolt** [comment]
@tabesin @meets2tarun  when did we shifted to master from generator branch for meilix generator and why ?

**Comment by pranav1698** [comment]
@xeon-zolt description is not provided, he must atleast fill the template with one or two lines also I don't think this will work, that's why I am asking for a test release,
That must be provided, otherwise their will be more issues rectifying the merged changes
I have always provided a test release whenever I have changed the settings of the ISO

File: scripts/chroot_reconst.sh (hunk 1 above)
Comment (alok760) [reply scripts/chroot_reconst.sh]: updated

**Comment by alok760** [comment]
@pranav1698 
heroku - https://meilix-gen.herokuapp.com/ (This points to the work branch of my fork - https://github.com/alok760/meilix)
release - https://github.com/alok760/meilix/releases/tag/wallpaperTest
travis for the above release - https://travis-ci.org/alok760/meilix/builds/508555643

ISO :
image

**Comment by xeon-zolt** [comment]
@alok760 next time please fill the form too

**Pull request merged**